### ISSUE TITLE
Support for Tabs while rendering text

### ISSUE THREAD

**Issue by tocsoft** [vidstige/NRasterizer#38]
calling `Renderer.Render(x, y, text, size. resolution)` doesn't support drawing tabs.

As a simple first pass it could be updated to just switch on tabs and advance `xx` by `n` spaces widths.

**Comment by vidstige** [comment]
Hmm, I wonder how this works with GDI+ and similar? You would like to advance to an the space character advance width times an even multiple of a tabstop, e.g. 4.

**Comment by tocsoft** [comment]
Yeah that make sense... recon the tab width should be definable with some configuration options passed into the `Renderer` instance, could even hoist size and resolution to class wide values with it.
